xodio/xod
# Table logs feature: collect, view and save tabular data

There is no issue 🙈 

- Adds new node `xod/debug/table-log`
- Adds new "Table Log" tab with table-view of the collected data

## How to test:
1. Place the new `xod/debug/table-log` node
2. Place `join` node
3. Set `\t` as a delimiter
4. Link some data sources (sensors / `system-time` / `sine-wave` / etc)
5. Link join with `table-log` node
6. Link pulse source to the `PUSH` pin of `table-log`
6. Run the program: simulate or upload with debugging
7. Double click on the `table-log` node to drill-down in the special tab "Table Log"

Also, you can
- place any amount of `table-log` nodes,
- send pulses to the `NEW` to create new sheets,
- `table-log` nodes might be placed inside other nodes,
- change node labels to make source names more readable
- run few different programs from one project and see logs of all runs
